docs/dependencies.rst

```diff
@@ -5,11 +5,11 @@ Xonsh currently has the following external dependencies,
 *Run Time:*
 
     #. Python v3.4+
-    #. PLY (optional, included with xonsh)
-    #. prompt-toolkit (optional)
-    #. Jupyter (optional)
-    #. setproctitle (optional)
-    #. distro (optional)
+    #. PLY (optional, included with xonsh, alternative parsing tool)
```

Comment:
PLY isn't an alternative parsing tool, it's just that we vend it ourselves in case it isn't installed locally.